app/models/peep.rb

```diff
@@ -0,0 +1,17 @@
+require 'data_mapper'
+require 'dm-postgres-adapter'
+
+class Peep
+
+include DataMapper::Resource
+
+property :author, String
```

Comment:
What if the author changes their name?  Shouldn't the `Peep` be related to the `User`?